app/routes/webhooks.py

```
@@ -14,74 +17,74 @@
-            last_name=user_data.last_name,
-            email=user_data.primary_email,
-            username=user_data.username,  # This is now cleaned by Pydantic
-            role="user",
-            active=True,
-        )
-        logger.info(f"Successfully created user: {user_data.id}")
-
-    except Exception as e:
-        logger.error(f"Failed to create user: {str(e)}")
-        raise
+	"""Handle user.created webhook event."""
```

Pylint (3.3.7)

[warning] Bad indentation. Found 1 spaces, expected 4

(W0311)